### PR TITLE
chore: add PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by Gautam-Arora24**
This Pull Request is intended to add PR template to the repo.

Comes in handy to guide contributors to fill PR descriptions to the point.

### CODE REVIEW

File: .github/issue_template/feature_request.md
Comment (arteevraina) [inline .github/issue_template/feature_request.md:9]: Hey @Gautam-Arora24 ! I think you can add a screenshot row in this template as well. But, please wait because maintainers might have a better approach for this. The same goes for the `Pull Request` template (for example: there might be a test case where UI is changed and sharing a screenshot in the PR Description is always welcomed. Happy Coding!

Comment (Gautam-Arora24) [reply .github/issue_template/feature_request.md:9]: Sure, I am waiting for the mentors' reply. They will guide me on how to proceed further :)

**Comment by DeltaHarbinger** [comment]
Hey @Gautam-Arora24, sorry for the long wait. I can review this and get back to you

**Comment by DeltaHarbinger** [comment]
@Gautam-Arora24 these templates are good, thanks for your contribution. These will be merged.

**Review by DeltaHarbinger — APPROVED** [review]

**Pull request merged**
